google-maps/src/main/java/org/odk/collect/googlemaps/GoogleMapFragment.java

```diff
@@ -84,6 +87,28 @@ public class GoogleMapFragment extends SupportMapFragment implements
     // Bundle keys understood by applyConfig().
     static final String KEY_MAP_TYPE = "MAP_TYPE";
 
+    public static boolean isAvailable(Context context) {
```

Comment:
Don't you think it would be better to keep these methods in a separate class than in the fragment?

Reply:
Ah yeah! This should probably get inlined into `GoogleMapConfigurator`.